includes/class-events.php

```diff
@@ -89,56 +91,41 @@ public function populate_concurrent_action_whitelist() {
 	}
 
 	/**
-	 * List events pending for the current period
+	 * List events pending for the current period.
 	 *
-	 * @param array $job_queue_size   Maximum number of events to return (excludes internal events).
-	 * @param array $job_queue_window How many seconds into the future events should be fetched.
-	 * @return array
+	 * @param null|int $job_queue_size   Maximum number of events to return (excludes internal events).
+	 * @param null|int $job_queue_window How many seconds into the future events should be fetched.
+	 * @return array Events to be run in the next batch.
 	 */
-	public function get_events( $job_queue_size = null, $job_queue_window = null ) {
+	public function get_events( $job_queue_size = null, $job_queue_window = null ): array {
```

Comment:
This is unfortunately named per what it does. As `Events/get_events()`, it actually more specifically fetches "due" events for the next round of cron running. Would be more aptly named `get_next_due_batch()` or something, so probably going to aim for some deprecation here in the future.